sdk/eventhub/event-hubs/samples/receiveEvents.ts

```diff
@@ -45,8 +48,8 @@ async function main() {
       resolve();
     }, 30000);
   });
+
```

Comment:
I just noticed, in the setTimeout call a few lines up you call close on subscription and consumerClient. Should we wrap that in a try/catch so we can reject the promise if either throws an error? Otherwise if either fails the promise will never settle.

Reply:
I've cleaned it up a bit - the setTimeout call wasn't super straightforward to understand anyways.